### DISCUSSION TITLE
is there a way to customize logos in quarto presentations?

### DISCUSSION THREAD

**Discussion by ameresv** [quarto-dev/quarto-cli#6635] [Q&A]
### Description

I would like to know if this can be done in Quarto presentation: Imagine you have two logos: logo1.png and logo2.png. The objective is using logo1.png in one slide and logo2.png in other slide, in addition I want to have a third slide with no logo at all. This is requested due to the slide backgrounds used to separate sections of the presentation. In that sense, is it possible to achieve this?
At this moment, I have tried this:

`# About me {background-color="black" .no-logo}`

or

`# About me {background-color="black"} {.no-logo}`

with an logo.css like this:

```
.reveal .slide-logo .no-logo {
  display: none !important;
}

section#no-logo {
  display: none !important;
}
```
Neither gave me any result. Any comment will be highly appreciated.
Warm regards

**Comment by mcanouil** [comment]
Could you share a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Comment by cderv** [comment]
For reference, cross posted at https://community.rstudio.com/t/is-there-a-way-to-customize-logos-in-quarto-presentations/172373

**Comment by mcanouil** [comment]
Related topics which include solutions to hide/unhide parts:

- https://github.com/quarto-dev/quarto-cli/discussions/557
- https://github.com/quarto-dev/quarto-cli/discussions/5566
